x-pack/solutions/observability/plugins/apm/public/components/app/metrics/index.tsx

```diff
@@ -38,13 +43,22 @@ export function Metrics() {
     return <ServerlessMetrics />;
   }
 
-  const hasStaticDashboard = hasDashboardFile({
-    agentName,
-    runtimeName,
-    serverlessType,
-  });
+  if (
+    !hasDashboard({ agentName, telemetrySdkName, telemetrySdkLanguage }) &&
```

Comment:
nit: you could call this `hasDashboard` only once, and use its return value here and for the `JsonMetricsDashboard` component

Reply:
Good point! I moved it to a `hasDashboardFile` and I am using the return value now